axon-server-connector/src/main/java/org/axonframework/axonserver/connector/event/axon/AxonServerEventStore.java

```diff
@@ -348,11 +362,12 @@ private void buildStorageEngine() {
                           "The PlatformConnectionManager is a hard requirement and should be provided");
 
             super.storageEngine(AxonIQEventStorageEngine.builder()
-                                                        .snapshotSerializer(snapshotSerializer.get())
-                                                        .upcasterChain(upcasterChain)
-                                                        .snapshotFilter(snapshotFilter)
-                                                        .eventSerializer(eventSerializer.get())
-                                                        .configuration(configuration)
+                    .snapshotSerializer(snapshotSerializer.get())
```

Comment:
I don't believe this change is correct, as I wouldn't expect the style guide to make it.